app/services/whatsapp/facebook_api_client.rb

```diff
@@ -0,0 +1,87 @@
+class Whatsapp::FacebookApiClient
+  include HTTParty
+  base_uri 'https://graph.facebook.com'
+
+  def initialize(access_token = nil)
+    @access_token = access_token
+    @api_version = GlobalConfigService.load('WHATSAPP_API_VERSION', 'v22.0')
+  end
+
+  def exchange_code_for_token(code)
+    response = self.class.get(
```

Comment:
HTTParty.get  instead of including it in the class, since thats the convention we follow.